recipe/post-link.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+NVCC_EXECUTABLE=$(which nvcc || exit 0)
+HOST_COMPILER_EXECUTABLE=$CXX
+if [[ -n "$NVCC_EXECUTABLE" && -n "$HOST_COMPILER_EXECUTABLE" ]]
+then
+    CUDA_BIN_DIR=$(dirname $NVCC_EXECUTABLE)
+    echo "ln -s $HOST_COMPILER_EXECUTABLE $CUDA_BIN_DIR/g++"
```

Comment:
Would redirect any output like so per [the recommendation at the end of this doc page]( https://docs.conda.io/projects/conda-build/en/latest/resources/link-scripts.html )

```suggestion
    echo "ln -s $HOST_COMPILER_EXECUTABLE $CUDA_BIN_DIR/g++" >> "${PREFIX}/.messages.txt" 2>&1
```